docs/developers/celestia-node-key.mdx

```diff
@@ -218,7 +218,7 @@ The keyring supports the following backends:
     kwallet     Uses KDE Wallet Manager as a credentials management application.
     pass        Uses the pass command line utility to store and retrieve keys.
     test        Stores keys insecurely to disk. It does not prompt for a password to be unlocked
-                and it should be used only for testing purposes.
+                and it should be use only for testing purposes.
```

Comment:
i'd revert

Reply:
I'm keen on knowing where this lives in `celestia-node` so that I can edit the CLI menu too

Reply:
for now 637cf8cedeb96a2bffee0c066d16fa0c1b1e113d